pkg/api/query/v1.go

```diff
@@ -430,6 +441,9 @@ func (qapi *QueryAPI) queryRange(r *http.Request) (interface{}, []error, *api.Ap
 
 	qe := qapi.queryEngine(maxSourceResolution)
 
+	// Record the query range requested
```

Comment:
Minor nit: let's end comments with a full stop:
```suggestion
	// Record the query range requested.
```